src/commoncode/fileutils.py

```diff
@@ -25,6 +25,7 @@
 from __future__ import absolute_import
 from __future__ import print_function
 from __future__ import unicode_literals
+from shutil import Error
```

Comment:
please separate the stdlib imports from the future imports by a blank line.

Reply:
Yes,I can do after testing.